### DISCUSSION TITLE
SCNX Token wo ist der wie bekommt man den und was hat es damit auf sich? (bot von Github)

### DISCUSSION THREAD

**Discussion by Tizian-Maxime-Weigt** [ScootKit/CustomDCBot#92] [Questions]
SCNX Token wo ist der wie bekommt man den und was hat es damit auf sich? (bot von Github)

ich suche den SCNX Token den ich in die config eintragen kann was hat es damit auf sich und wo herbekomme ich diesen?

**Comment by SCDerox** [comment]
Sorry, but the official language of this repository is English. Please keep your discussions in English. The SCNX Token won't be used unless the bot runs in our production SCNX environment. You don't need to specify it. The token is used to allow encrypted communication between hosted SCNX-Bots and the SCNX-Dashboard. It's used to allow Login-As-A-Bot, Issue-Reporting, Custom-Commands and more (all of these features are only available in the hosted version).

**Reply by Tizian-Maxime-Weigt** [reply]
wäre es den möglich das jemand aus dem team mir von meinem bot den token sendet oder ist sowas team intern ?

**Reply by SCDerox** [reply]
Sorry, but we are unable to assist you, if you don't speak English. The official language of this repository is English. Please keep your discussions in English. I already answered your question in my last answer.
